keel-web/src/test/kotlin/com/netflix/spinnaker/keel/rest/AdminControllerTests.kt

```diff
@@ -17,6 +17,7 @@ import org.springframework.test.web.servlet.MockMvc
 import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers.content
 import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+import java.time.Instant
```

Comment:
If you're mocking all the data, what value do you expect to get out of these tests? I feel like we don't need to test our object to json translation for every controller. Are you explicitly cutting out data from the kotlin object and testing that?

Reply:
Yeah, good question... I landed here while looking for repository tests that read the data, which I wanted to extend to make sure the parsing of the ULID is working as expected, but there are none as far as I can see, so I ended up just adding this here which, you're right, doesn't test anything useful, instead of writing the repository tests. In other words, I did something stupid. 😆  Will add a repository test!

Reply:
okay cool, haha. I generally don't find a lot of value in controller tests (and find them hard to update) so I try to question them every chance I get. If you feel like this test might not be useful, feel free to delete it!